include/boost/gil/image_processing/numeric.hpp

```diff
@@ -48,6 +50,60 @@ inline double lanczos(double x, std::ptrdiff_t a)
     return 0;
 }
 
+/// \brief Generate mean kernel
+/// \ingroup ImageProcessingMath
+///
+/// Fills supplied view with normalized mean
+/// in which all entries will be equal to
+/// 1 / (dst.size())
+void generate_normalized_mean(boost::gil::gray32f_view_t dst) {
```

Comment:
The full qualification w/ namespace `boost::gil::gray32f_view_t` is not required, or is it?

Reply:
It's just that I'm paranoid about ADL, even though they apply only to function calls. I will deal with the inlines shortly, thanks for catching that.